### PR TITLE
festival LP - full article on new window

### PR DESCRIPTION
**Pull request by Poppe-A**

### CODE REVIEW

File: apps/festival/festival/src/app/landing/landing.component.html [hunk 1]
```diff
@@ -121,17 +121,19 @@ <h3>{{opinion.name}}</h3>
...
       <ng-container *ngFor="let review of reviews">
-        <mat-card fxLayout="column center" fxFlexAlign="stretch"> 
-          <img mat-card-image [asset]="review.img" alt="review image">
-          <div fxLayout="column" fxLayoutAlign="space-between" fxLayoutGap="24px" fxFlex>
-            <mat-card-title>{{review.title}}</mat-card-title>
-            <mat-card-content>{{review.text}}</mat-card-content>
-            <div fxLayout fxLayoutAlign="center center" fxFlexAlign="end" fxLayoutGap="16px">
-              <span class="mat-caption">From</span>
-              <img [asset]="review.logo">
+        <a href="{{review.link}}" target="_blank" fxLayout fxFlexAlign="stretch">
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/landing/landing.component.html:124]: Do you really need `fxLayout` and `fxFlexAlign="stretch` here? I don't know but maybe `display: block;` or something would be enough?

**Review by RemcoSimonides — APPROVED** [review]

File: apps/festival/festival/src/app/landing/landing.component.html (hunk 1 above)
Comment (Poppe-A) [reply apps/festival/festival/src/app/landing/landing.component.html:124]: Without "stretch", the 3 cards wouldn't have the same height

Comment (RemcoSimonides) [reply apps/festival/festival/src/app/landing/landing.component.html:124]: What about a `height: 100%`?

Comment (Poppe-A) [reply apps/festival/festival/src/app/landing/landing.component.html:124]: I tried but it doesn't work

**Pull request merged**